admin/src/seeds/index.ts

```diff
@@ -1,19 +1,23 @@
+import type { Banner, Event, Space } from '@phits-tech/common/dao-firestore'
 import { BANNERS, EVENTS, SPACES } from '@phits-tech/common/dao-firestore'
 
+import { productionWarning } from '@/_services/modes'
 import { context } from '~/context'
 import { MODE } from '~/modes'
 import migrate from '../migrations/migrate'
 
-import { banners } from './data/banners'
-import { events } from './data/events'
-import { spaces } from './data/spaces'
-
 const db = context.db
 
 const main = async (): Promise<void> => {
-  if (MODE !== 'emu') return console.warn('This script is exclusively for local testing')
+  if (!process.argv.includes('--yes')) {
+    await productionWarning(__filename)
+  }
   await migrate()
 
+  const banners = await import(`./${MODE}/banners`).then(_ => _.banners).catch(_ => []) as Banner[]
+  const events = await import(`./${MODE}/events`).then(_ => _.events).catch(_ => []) as Event[]
+  const spaces = await import(`./${MODE}/spaces`).then(_ => _.spaces).catch(_ => []) as Space[]
+
```

Comment:
I'm not sure about `_ => _.banners`... is poor practice in JS/TS?

Reply:
I think it's poor practice to name something `_` and then use it!! Underscore is typically reserved to mean "I don't use this variable".

I see you problem; the data files export named consts, so you have to reference them. Another option would be to refactor to use a default export:
```
const banners: Banner[] = [...]
export default banners
```
Then you could delete the `then` entirely.

Reply:
Done